pyproject.toml

```
@@ -16,17 +16,19 @@ classifiers = [
     "Framework :: Django :: 3.2",
     "Framework :: Django :: 4.0",
     "Framework :: Django :: 4.1",
+    "Framework :: Django :: 5.0",
```

Missing a version in here:

```suggestion
    "Framework :: Django :: 4.2",
    "Framework :: Django :: 5.0",
```